rakelib/examples_config.yml

```diff
@@ -0,0 +1,16 @@
+example_type_data:
+  -
+    name: basic
```

Comment:
@justin808 I was thinking of keeping this file separate and then just making another yml file for the apps we generate for the diffs repo. These apps are different than those because they are going to have noise from the dev_tests generator and are within the gem's git repo.